### ISSUE TITLE
rounded-common.rasi not found.

### ISSUE THREAD

**Issue by jackdrowlands** [lr-tech/rofi-themes-collection#17]
Whenever I tried setting a rounded-color theme, an error would occur stating:
```
Failed to load theme. Try to load default: 
Failed to open theme: <i>/home/name/rofi-themes-collection/rounded-common.rasi</i>
Error: <b>No such file or directory</b>
```
This was fixed by moving this file to the expected path, however, whenever I ran rofi within my user directory, it would default to the default, with an error stating:
```
Failed to load theme. Try to load default: 
Failed to open theme: <i>/home/name/rounded-common.rasi</i>
Error: <b>No such file or directory</b>
```
I managed to fix this error by modifying the rounded-color-dark.rafi. I changed the import statement to include the full path, which seemed to fix the error.
It is all working now that I have made that change.

**Comment by aVastAbyss** [comment]
I got a very similar error as the one you got and was _almost_ able to fix it by moving rounded-common.rasi to the expected directory, but I got the following error:
```
Failed to open theme: /home/name/.local/share/rofi/themes/rounded-common.rasi
Error: Too many open files
```
I'll update this message when I get it resolved.  In the meantime, let me know if you have any thoughts.

**Edit:**
The issue was a user error.  I somehow managed to overwrite the contents of rounded-common.rasi with the contents of rounded-gray-dark.rasi.  This meant that when rounded-gray-dark.rasi called rounded-common.rasi, rounded-common.rasi would call itself recursively, resulting in the process quickly reaching the max limit number of files a single process is allowed to open (1024).

Issue resolved.
